enterprise_access/apps/api_client/lms_client.py

```diff
@@ -115,6 +127,52 @@ def unlink_users_from_enterprise(self, enterprise_customer_uuid, user_emails, is
             logger.exception(msg, enterprise_customer_uuid)
             raise
 
+    def fetch_group_members(
+        self,
+        group_uuid,
+        sort_by=None,
+        user_query=None,
+        fetch_removed=False,
+        is_reversed=False,
+        page=None,
+    ):
+        """
+        Fetches enterprise group member records from edx-platform.
+
+        Params:
+        - ``sort_by`` (string, optional): Specify how the returned members should be ordered. Supported sorting values
+        are `member_details`, `member_status`, and `recent_action`.
+        - ``user_query`` (string, optional): Filter the returned members by user email with a provided sub-string.
+        - ``fetch_removed`` (bool, optional): Include removed membership records.
+        - ``is_reversed`` (bool, optional): Reverse the order of the returned members.
+        - ``page`` (int, optional): Which page of paginated data to return. If no page is supplied, fetch all records.
+        """
+        group_members_url = self.enterprise_group_members_endpoint(group_uuid)
+        params = {
+            "sort_by": sort_by,
+            "user_query": user_query,
+            "fetch_removed": fetch_removed,
+            "is_reversed": is_reversed,
+        }
+        if page:
+            params['page'] = page
+
+        response = self.client.get(group_members_url, params=params)
+        response.raise_for_status()
+        response_json = response.json()
+        results = response_json.get('results', [])
+        if not page:
+            next_page = response.json().get("next")
+            while next_page:
```

Comment:
ah, I see. I would make this more explicit by allowing an optional `traverse_pagination` parameter of this method, and maybe the view. Presumably our frontends won't actually want to kick off a loop of requests via the backend.